packages/editor-ui/src/components/Node/NodeCreator/Modes/NodesMode.vue

```diff
@@ -56,10 +56,12 @@ function onSelected(item: INodeCreateElement) {
 	if (item.type === 'subcategory') {
 		const subcategoryKey = camelCase(item.properties.title);
 		const title = i18n.baseText(`nodeCreator.subcategoryNames.${subcategoryKey}` as BaseTextKey);
+		const info = i18n.baseText(`nodeCreator.subcategoryInfos.${subcategoryKey}` as BaseTextKey);
```

Comment:
We should only set it if info locale exists, otherwise leads to: 
![CleanShot 2024-07-24 at 17 24 09](https://github.com/user-attachments/assets/377db38b-556a-4b51-9856-f4897c46beeb)
![CleanShot 2024-07-24 at 17 23 49](https://github.com/user-attachments/assets/92857750-8091-4f4a-909c-398922326c4a)

Reply:
Fixed this, thanks!